pgactivity/cli.py

```diff
@@ -2,104 +2,103 @@
 import socket
 import sys
 import time
-from optparse import OptionParser, OptionGroup
+from argparse import ArgumentParser
 
 from blessed import Terminal
 from psycopg2.errors import OperationalError
 
 from . import __version__, data, types, ui
 
 
-def get_parser() -> OptionParser:
-    parser = OptionParser(
-        add_help_option=False,
-        version="%prog " + __version__,
-        usage="%prog [options] [connection string]",
+def get_parser() -> ArgumentParser:
+    parser = ArgumentParser(
+        usage="%(prog)s [options] [connection string]",
+        description=(
+            "htop like application for PostgreSQL " "server activity monitoring."
+        ),
         epilog=(
             "The connection string can be in the form of a list of "
             "Key/Value parameters or an URI as described in the PostgreSQL documentation. "
             "The parsing is delegated to the libpq: different versions of the client library "
             "may support different formats or parameters (for example, connection URIs are "
             "only supported from libpq 9.2)"
         ),
-        description=(
-            "htop like application for PostgreSQL " "server activity monitoring."
-        ),
+        add_help=False,
+    )
+    # Connection string
+    parser.add_argument(
+        "connection_string",
```

Comment:
I'd suggest to add `metavar="connection string"` so that the help of this argument will be consistent with the usage line.

Reply:
done